Localization.EntityFramework/Startup.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Features;
+using Microsoft.AspNet.Localization;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+
+namespace EFLocalizationSample
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSingleton<IStringLocalizerFactory, EFStringLocalizerFactory>();
+        }
+
+        public void Configure(IApplicationBuilder app, IStringLocalizerFactory factory)
```

Comment:
Change param from `factory` to something more specific, e.g. `localizerFactory`.